### ISSUE TITLE
Cache uploaded media

### ISSUE THREAD

**Issue by bharathbrat** [zhukov/webogram#712]
After sending any media file on a group, if you open the file it downloads again. Would it not be better if can cache this. It could be as simple as storing the path to file. So at least at your view there will be no requirement to download the file again. I could try to do this. If you guys think it is required.

**Comment by Walkman100** [comment]
This happens on Telegram desktop too, not sure but I think Telegram Android app remembers that you uploaded it.

**Comment by bharathbrat** [comment]
Yes. I observed this only on the desktop version. I use webogram on firefox, fyi.

**Comment by Walkman100** [comment]
Desktop version of Webogram, or Telegram Desktop?
